MvvmCross-iOSSupport/MvvmCross.iOS.Support.XamarinSidebar/MvxSidebarPresenter.cs

```diff
@@ -33,94 +32,77 @@ protected virtual bool PresentationHintHandler(MvxPanelPresentationHint hint)
             return true;
         }
 
-        public override void Show(MvxViewModelRequest request)
+        protected override void RegisterAttributeTypes()
         {
-            IMvxIosView viewController = Mvx.Resolve<IMvxIosViewCreator>().CreateView(request);
-            Show(viewController);
+            base.RegisterAttributeTypes();
+
+            _attributeTypesToShowMethodDictionary.Add(
+                typeof(MvxSidebarPresentationAttribute),
+                (vc, attribute, request) => ShowSidebarViewController(vc, attribute as MvxSidebarPresentationAttribute, request));
```

Comment:
Hmm i don't see how that could happen. The base presenter also does it this way.